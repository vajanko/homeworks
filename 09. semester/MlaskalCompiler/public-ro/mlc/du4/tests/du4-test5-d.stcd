<?xml version="1.0" encoding="UTF-8"?>
<symbol_tables>
	<types>
		<BOOLEAN type="T0"/>
		<INTEGER type="T1"/>
		<REAL type="T2"/>
		<STRING type="T3"/>
	</types>
	<labels>
	</labels>
	<symbols>
		<SKIND_TYPE name="BOOLEAN" type="T0"/>
		<SKIND_CONST name="FALSE" value="0" type="T0"/>
		<SKIND_FUNCTION name="FNC2" type="T?">
			<parameters>
			</parameters>
			<labels>
			</labels>
			<symbols>
			</symbols>
		</SKIND_FUNCTION>
		<SKIND_TYPE name="INTEGER" type="T1"/>
		<SKIND_PROCEDURE name="PROC2">
			<parameters>
			</parameters>
			<labels>
			</labels>
			<symbols>
			</symbols>
		</SKIND_PROCEDURE>
		<SKIND_FUNCTION name="READI" type="T1">
			<parameters>
			</parameters>
			<labels>
			</labels>
			<symbols>
			</symbols>
		</SKIND_FUNCTION>
		<SKIND_FUNCTION name="READR" type="T2">
			<parameters>
			</parameters>
			<labels>
			</labels>
			<symbols>
			</symbols>
		</SKIND_FUNCTION>
		<SKIND_FUNCTION name="READS" type="T3">
			<parameters>
			</parameters>
			<labels>
			</labels>
			<symbols>
			</symbols>
		</SKIND_FUNCTION>
		<SKIND_TYPE name="REAL" type="T2"/>
		<SKIND_TYPE name="STRING" type="T3"/>
		<SKIND_CONST name="TRUE" value="1" type="T0"/>
		<SKIND_PROCEDURE name="WRITEI">
			<parameters>
				<PMODE_BY_VALUE name="P" type="T1"/>
			</parameters>
			<labels>
			</labels>
			<symbols>
			</symbols>
		</SKIND_PROCEDURE>
		<SKIND_PROCEDURE name="WRITELN">
			<parameters>
			</parameters>
			<labels>
			</labels>
			<symbols>
			</symbols>
		</SKIND_PROCEDURE>
		<SKIND_PROCEDURE name="WRITER">
			<parameters>
				<PMODE_BY_VALUE name="P" type="T2"/>
			</parameters>
			<labels>
			</labels>
			<symbols>
			</symbols>
		</SKIND_PROCEDURE>
		<SKIND_PROCEDURE name="WRITES">
			<parameters>
				<PMODE_BY_VALUE name="P" type="T3"/>
			</parameters>
			<labels>
			</labels>
			<symbols>
			</symbols>
		</SKIND_PROCEDURE>
	</symbols>
</symbol_tables>
