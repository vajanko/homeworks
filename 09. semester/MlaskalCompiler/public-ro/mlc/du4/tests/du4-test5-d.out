Error (line 3): 'INT' is not a type
There were some errors 1 and warnings 0
